scania-digital-design-system/sdds
# feat(button): migrate button component

**Describe pull-request**  
Migration of the button component, both native and webcomponent

**Solving issue**  
Fixes: [AB#2398](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2398)

**How to test**  
_Add description how to test if possible_
1. Go to storybook link below
2. Check in components -> button
3. Check that the design is aligned with the one in Figma.

**Screenshots**  
-

**Additional context**  
-
